internal/agent/device/status/config_hooks.go

```diff
@@ -25,7 +26,7 @@ func newHooks(log *log.PrefixLogger, manager hook.Manager) *Hooks {
 
 // Export returns the status of the config hooks.
 func (s *Hooks) Export(ctx context.Context, status *v1alpha1.DeviceStatus) error {
-	return nil
+	return errors.Join(s.manager.Errors()...)
```

Comment:
question: since this is general status do we need to prefix these errors with any context or will the errors have enough details for a user to understand already?

Reply:
Will modify to make sure